Added fantasy-name-gen plugin

ringmaster opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin: https://github.com/ringmaster/fantasty-name-gen/tree/master

## Release Checklist
- [x] I have tested the plugin on
  - [ ]  Windows
  - [x]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
  - [ ] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the tips in https://github.com/obsidianmd/obsidian-releases/blob/master/plugin-review.md and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

joethei commented:
Since you don't use settings, you can remove any references to them.

[id: 'sample-editor-command',](https://github.com/ringmaster/fantasty-name-gen/blob/6d90f3c6837c3c6c7b9473c82233a7d04b6d4fda/main.ts#L56)
This should be a meaningful id, Obsidian will automatically prefix the command id with the plugin id, you don't need to include it yourself.

ringmaster commented:
I've updated the plugin with a unique id, removed the settings functions, and created a new release with an updated patch version of 1.0.1.  Thanks for the review!
